File: 4D4Progr4m5/SelectionSort.c
#include<stdio.h>
int main()
{
    int a[100];
    int i, j, n, temp;
    printf("Enter the number of elements: ");
    scanf("%d",&n);
    printf("Enter the elements into the array:\n");
    for(i=0;i<n;i++)
    scanf("%d",&a[i]);
    printf("Array before sorting:\n");
    for(i=0;i<n;i++)
    printf(" %d ",a[i]);

    for(i=0;i<n-1;i++)
    {
        for(j=i+1;j<n;j++)
        {
            if(a[i]>a[j])
            {
                temp=a[j];
                a[j]=a[i];
                a[i]=temp;
            }
        }
    }
    printf("\nArray after sorting:\n");
    for(i=0;i<n;i++)
    printf(" %d ",a[i]);
    return 0;

}

File: 4D4Progr4m5/SelectionSort.py
n = int(input("Enter the number of elements: "))
a=input("Enter the elements seperated by space:")
Array = [int(x) for x in a.split()]
print("Unsorted Array is:")
print(Array)
for i in range(0,len(Array)-1):
    for j in range(i+1,len(Array)):
        if(Array[i]>Array[j]):
            temp=Array[i]
            Array[i]=Array[j]
            Array[j]=temp
print("Sorted Array is:")
print(Array)
